docs/use_notary.md

```diff
@@ -24,3 +24,12 @@ Because by default the local directory for storing meta files for notary client
 alias notary="notary -s https://192.168.0.5:4443 -d ~/.docker/trust --tlscacert  /etc/docker/certs.d/192.168.0.5/ca.crt"
 
 ```
+
+### Remove Tag Signature In Docker Content Trust
+
+After alias for notary was set, you can use remove commond to remove a tag signature:
```

Comment:
please change to 'After alias for Notary is set, you can use 'remove' command to remove a tag signature:'